docs/simulators.rst

```diff
@@ -16,6 +16,36 @@ See the individual entries for details on each interface's capabilities and how
 Currently Supported
 ===================
 
+MetaDrive (Optional)
+----------------------------
+
+Scenic supports integration with the `MetaDrive <https://metadriverse.github.io/metadrive/>`_ simulator as an optional dependency,
+enabling users to describe dynamic simulations of vehicles, pedestrians, and traffic scenarios.
+If your system supports it, you can install it with:
+
+.. code-block:: console
+
+    pip install scenic[metadrive]
```

Comment:
```suggestion
    python -m pip install scenic[metadrive]
```
This is what we do in the Scenic installation instructions -- it's a little safer since sometimes virtual environments don't capture the `pip` command but only `python` (for reasons I don't understand).